udata_hydra_csvapi/utils.py

```diff
@@ -0,0 +1,37 @@
+def build_sql_query_string(request_arg: list, page_size: int, offset: int = 0) -> str:
+    sql_query = []
+    for arg in request_arg:
+        argument, value = arg.split("=")
+        if "__" in argument:
+            column, comparator = argument.split("__")
```

Comment:
What if a column has a `__`? It's the case of the `__id` column in the sample data.
I don't think there is any encoding of `_`, is there? We could just document that we can't query on these columns, but we should make it explicit :)